include/odp/api/spec/schedule_types.h

```diff
@@ -303,6 +536,15 @@ typedef struct odp_schedule_config_t {
 		 */
 		odp_bool_t worker;
 
+		/** Parameters for ODP_SCHED_GROUP_ALL schedule group */
+		odp_schedule_group_param_t all_param;
+
+		/** Parameters for ODP_SCHED_GROUP_CONTROL schedule group */
+		odp_schedule_group_param_t control_param;
+
+		/** Parameters for ODP_SCHED_GROUP_WORKER schedule group */
+		odp_schedule_group_param_t worker_param;
```

Comment:
As discussed, we need to allow calling at least `odp_schedule_min_prio()` and `odp_schedule_max_prio()`/`odp_schedule_num_prio()` before `odp_schedule_config()` to enable filling per priority specific cache stashing hints.

Reply:
Added a patch to state this.

Reply:
As discussed offline, dropped the latest patch v8, minimum priority will be part of capabilities.